tests/RequestTest.php

```diff
@@ -196,4 +196,14 @@ public function testGetAccept()
         $this->assertEquals('accept', $this->request->getAccept('default'));
     }
 
+    public function testContentRange()
+    {
```

Comment:
Let's add more assertions, including ones that should fail.

Reply:
done